MobileWalletTests/MobileWalletTests.swift

```diff
@@ -91,7 +91,31 @@ class MobileWalletTests: XCTestCase {
             XCTFail("Failed to iterate through theme assets")
         }
     }
+    
+    func testDateFallsInWeek() {
+        let testDateString = "2019-10-01"
+        let dateFormatter = DateFormatter()
+        dateFormatter.dateFormat = "yyyy-mm-dd"
+        guard let testDate = dateFormatter.date(from: testDateString) else {
+            XCTFail("Failed to get the date from the string value")
+            return
+        }
+        
+        if DateConfig.dateFallsInCurrentWeek(date: testDate) == true {
```

Comment:
I think this test will fail in a few days, `2019-10-01` will soon not be in the current week. Maybe adjust `dateFallsInCurrentWeek ` to accept 2 dates to compare if they're in the same week. Then in the implementation send through the current date as the second arg.

Reply:
Ah! Good point.